### DISCUSSION TITLE
Windows installation error

### DISCUSSION THREAD

**Discussion by jarvisx17** [juanmc2005/diart#151] [Q&A] [answered]
ImportError: FFmpeg libraries are not found. Please install FFmpeg.

I tried all possible ways to install ffmpeg lib in my windows but it still showing above error in my windows system.

**Comment by juanmc2005** [comment]
Hi @jarvisx17, did you install with conda?

```conda install portaudio pysoundfile ffmpeg -c conda-forge```

**Reply by jarvisx17** [reply]
yes i tried but still same error.

ImportError                               Traceback (most recent call last)
Cell In[5], line 3
      1 import torchaudio
      2 print(torchaudio._extension._FFMPEG_INITIALIZED) #False
----> 3 torchaudio._extension._init_ffmpeg()

File [c:\Users\jigpa\OneDrive\Desktop\Diart-conda\.conda\lib\site-packages\torchaudio\_extension.py:73](file:///C:/Users/jigpa/OneDrive/Desktop/Diart-conda/.conda/lib/site-packages/torchaudio/_extension.py:73), in _init_ffmpeg()
     71     _load_lib("libtorchaudio_ffmpeg")
     72 except OSError as err:
---> 73     raise ImportError("FFmpeg libraries are not found. Please install FFmpeg.") from err
     75 import torchaudio._torchaudio_ffmpeg  # noqa
     77 torch.ops.torchaudio.ffmpeg_init()

ImportError: FFmpeg libraries are not found. Please install FFmpeg.

**Reply by juanmc2005** [reply]
Did you try asking the torchaudio community? It looks like the library itself isn't recognizing your ffmpeg installation.

**Comment by jarvisx17** [comment, marked as answer]
conda install 'ffmpeg<4.4'

This installation helps to work successfully in windows.
